src/backend/identity/index.ts

```diff
@@ -35,22 +35,23 @@ export interface User {
 }
 
 /**
- * Defines a notification to send to a user’s Koji account.
- * Send notifications with [[pushNotificationToOwner]], for the user who created the Koji, or [[pushNotificationToUser]], for a user who interacts with the Koji and has granted the appropriate authorization.
+ * Defines a notification to send to a user account.
+ * Use [[pushNotificationToOwner]] to send notifications to the creator who posted the Koji app in their profile.
```

Comment:
```suggestion
 * Use [[pushNotificationToOwner]] to send notifications to the creator of the app.
```
Technically, they don't need to post it to their profile, so we can simplify this language a bit here.

Reply:
Confusion between "developer" and "creator" here.
My understanding is that the "developer" forks a repository to modify the code to create a new app. On the other hand, a creator only customizes (or creates custom versions) of the app by changing its settings or metadata and/or by replacing the app's assets. The creator does not touch the code directly.

Reply:
Replaced with "owner".